.circleci/config.yml

```diff
@@ -120,13 +120,13 @@ workflows:
           file: mapping/staging/mapping.txt
           app: $APP_CENTER_APP_NAME
           token: $APP_CENTER_TOKEN
-          version-name: $(grep "VERSION_NAME" testapp/build/generated/source/buildConfig/staging/com/rakuten/tech/mobile/miniapp/testapp/BuildConfig.java | awk '{print $7}' | tr -d \''";\')
+          version-name: echo $(grep "VERSION_NAME" testapp/build/generated/source/buildConfig/staging/com/rakuten/tech/mobile/miniapp/testapp/BuildConfig.java | awk '{print $7}' | tr -d '";')
           version-code: 1
           requires:
             - android-sdk/build
           filters:
             branches:
-              only: master
+              only: /.*/
```

Comment:
do you nee a filter if you want all branches? :)

Reply:
sorry, I temporary enabled it because of checking something in circleci in this branch. this is a redundant change.